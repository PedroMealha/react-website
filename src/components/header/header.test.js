import { shallow } from "enzyme";
import Header from "./header.component";

describe('Header component', () => {
	it('expect to render Header component', () => {
		expect(shallow(<Header section='about' />)).toMatchSnapshot();
	})
})