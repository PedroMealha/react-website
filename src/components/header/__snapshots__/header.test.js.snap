// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component expect to render Header component 1`] = `
<div
  className="header"
>
  <div
    className="gradient"
  />
  <div
    className="copy"
  >
    <h1>
      about me
    </h1>
    <h3>
      Who am I?
    </h3>
  </div>
</div>
`;
